spec/factories/collections_factory.rb

```diff
@@ -1,5 +1,8 @@
 FactoryBot.define do
   factory :collection do
+    # DEPRECATION: This factory is being replaced by collection_lw defined in collections.rb.  New tests should use the
+    # light weight collection factory.  DO NOT ADD tests using this factory.
+    #
```

Comment:
I'm not sure of how deprecations are handled b/c i've never dealt with it, but should we do something like this so the testmaker sees the warning?

`warn "[DEPRECATION] :collection is deprecated, please use :collection_lw instead, located in spec/factories/collections.rb"`

Reply:
I stuck that in there for myself and failed to update it to an official deprecation warning.  Thanks for catching that.

Reply:
I updated to have a deprecation warning, but there are still many many tests using the original factory.  It really clogs up the test output.  I wonder if we could keep this as a comment until more of the tests are converted over to the new factory.  I'll bring this up at tech call to get a feel for how folks prefer to handle this.

Reply:
What do you think about a notice at the top saying something like...

```
# WILL BE DEPRECATED: This factory is being replaced by collection_lw defined in collections.rb.  New tests should use the
# light weight collection factory.  DO NOT ADD tests using this factory.
```
I definitely didn't like having deprecate warning on.  There are too many places where it is still in use within our own code.

Reply:
I think we definitely want a "do not use" statement. But no noisy deprecation warning.